src/python/platforms/android/linkify.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Used to download Android symbols."""
+from __future__ import print_function
+
+import html
+
+from base import utils
+from build_management import source_mapper
+
+
+def linkify_kernel_stack_frame(stack_frame, kernel_prefix, kernel_hash):
```

Comment:
After addressing the comment above, this function becomes 
get_kernel_stack_frame